packages/image_picker/image_picker_ios/ios/Classes/FLTPHPickerSaveImageToPathOperation.m

```diff
@@ -184,4 +187,39 @@ - (void)processImage:(NSData *)pickerImageData API_AVAILABLE(ios(14)) {
   }
 }
 
+/**
+ * Processes the video.
+ */
+- (void)processVideo API_AVAILABLE(ios(14)) {
+  NSString *typeIdentifier = self.result.itemProvider.registeredTypeIdentifiers.firstObject;
+  [self.result.itemProvider
+      loadFileRepresentationForTypeIdentifier:typeIdentifier
+                            completionHandler:^(NSURL *_Nullable videoURL,
+                                                NSError *_Nullable error) {
+                              if (videoURL == nil) {
+                                return;
+                              }
+                              NSString *fileName = [videoURL lastPathComponent];
+                              NSURL *destination = [NSURL
+                                  fileURLWithPath:[NSTemporaryDirectory()
+                                                      stringByAppendingPathComponent:fileName]];
+
+                              if ([[NSFileManager defaultManager]
+                                      isReadableFileAtPath:[videoURL path]]) {
+                                NSError *error;
+                                if (![[videoURL path] isEqualToString:[destination path]]) {
+                                  [[NSFileManager defaultManager] copyItemAtURL:videoURL
+                                                                          toURL:destination
+                                                                          error:&error];
+                                  if (error) {
+                                    if (error.code != NSFileWriteFileExistsError) {
+                                      return;
```

Comment:
Similar to above comment about errors. I don't know if completely ignoring them is the right move. It's debatable if we should return the error since this is multi-select, but I would think we should at least print out the error. What do you think?